packages/cloud2edge/values.yaml

```diff
@@ -11,8 +11,9 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
+tenant: "org.eclipse.packages.c2e"
 demoDevice:
-  tenant: "org.eclipse.packages.c2e"
+  enabled: true
```

Comment:
Since there is also an `enabled` field in Hono device data, I think it would be better to choose a different property name here (e.g. `skip` or `omit`).